src/clarity-angular/utils/_variables.clarity.scss

```diff
@@ -116,10 +128,10 @@ $clr-h3:(
     color: $clr-black,
     font-weight: $clr-typography-lightfont,
     font-family: $clr-altFont,
-    font-size: rem(22/$clr-rem-denominator),
+    font-size: 0.916667rem,
```

Comment:
So with values that not exactly divisible by 24px, we need atleast precision upto 5 decimal places? Just checking with you so that I remember when such cases arise later on.

Reply:
It's 6 decimal places. That's the most the best browser went out to. Some browsers (like Edge) appeared to only go out to 2 decimal places (and didn't round up either).